src/components/ConfirmationGraphic/index.tsx

```diff
@@ -6,6 +6,7 @@ import { Icon, LogoNames } from "@nypl/design-system-react-components";
 import useFormDataContext from "../../context/FormDataContext";
 
 const ConfirmationContainer: React.FC = () => {
+  const canvasArgs = { role: "img", ["aria-label"]: "Scannable barcode" };
```

Comment:
The `canvas` renders an image so it should be treated as one. Instead of an `alt`, it should have an `aria-label`. This I'm not too sure why but the eslinter and jest-axe both complained.